Library/Formula/youtube-dl-gui.rb

```diff
@@ -0,0 +1,53 @@
+class YoutubeDlGui < Formula
+  desc "A cross platform front-end GUI of the popular youtube-dl."
+  homepage "https://github.com/MrS0m30n3/youtube-dl-gui"
+  url "https://github.com/MrS0m30n3/youtube-dl-gui.git",
+    :revision => "95189d3d82c159e069697fa65058c8a9f88ff686"
```

Comment:
A new release will be available after closing https://github.com/MrS0m30n3/youtube-dl-gui/issues/39
The time depends on how fast i am going to get a reply on https://github.com/rg3/youtube-dl/issues/6119